README.md

```diff
@@ -195,6 +195,28 @@ Authorization in `root:redhat-appstudio` and `root:redhat-hacbs` workspaces in C
 
 For access to the OpenShift staging cluster, the user must be added to the `stage` team in the `redhat-appstudio-sre` Github organization.
 
+## Enabling Monitoring for workload cluster
+
+Prometheus Operator is deployed through ArgoCD. This is configure to run in `appstudio-workload-monitoring` namespace. 
+This prometheus is accessed through exported route, which is secured by github oauth. Github oauth secrets are configured in the cluster through secret
```

Comment:
I wanted to add the web access route in another PR after checking the generated service name and adding github oauth. let me add it here.

Reply:
if this can help, you'll need something like this:
```
---
# Route and service to provide a secured access to the Prometheus console
# via the oauth2-proxy sidecar (ie, that's the route for the human beings)
apiVersion: route.openshift.io/v1
kind: Route
metadata:
  namespace: appstudio-workload-monitoring
  name: prometheus
  annotations:
    kubernetes.io/tls-acme: "true"
spec:
  to:
    kind: Service
    name: prometheus-oauth2
  port:
    targetPort: oauth2-proxy
  tls:
    insecureEdgeTerminationPolicy: Redirect
    termination: reencrypt
--- 
apiVersion: v1
kind: Service
metadata:
  namespace: appstudio-workload-monitoring
  name: prometheus-oauth2
  labels:
    app.kubernetes.io/instance: prometheus
  annotations:
    service.alpha.openshift.io/serving-cert-secret-name: prometheus-oauth2-tls
spec:
  ports:
  - name: oauth2-proxy
    port: 9091
    protocol: TCP
    targetPort: 9091
  selector:
    app.kubernetes.io/instance: prometheus
---
# Route and service to provide a secured access to the Prometheus query/metrics endpoint
# via the oauth-proxy sidecar (ie, that's the route for Grafana with a Service Account token)
apiVersion: route.openshift.io/v1
kind: Route
metadata:
  namespace: appstudio-workload-monitoring
  name: prometheus-oauth
  annotations:
    kubernetes.io/tls-acme: "true"
spec:
  to:
    kind: Service
    name: prometheus-oauth
  port:
    targetPort: oauth-proxy
  tls:
    termination: reencrypt
---
apiVersion: v1
kind: Service
metadata:
  namespace: appstudio-workload-monitoring
  name: prometheus-oauth
  labels:
    app.kubernetes.io/instance: prometheus
  annotations:
    service.alpha.openshift.io/serving-cert-secret-name: prometheus-oauth-tls
spec:
  ports:
  - name: oauth-proxy
    port: 9092
    protocol: TCP
    targetPort: 9092
  selector:
    app.kubernetes.io/instance: prometheus
```

Reply:
yeah already added that. thanks